### PR TITLE
[NMS] Add APN create/Edit Dialog

### LINKED ISSUE #2263: [nms] ApnProfile creation/edit interface

### LINKED ISSUE #2764: Add/Edit APN dialog
https://www.figma.com/file/hw8iheccicYs6tW7gsFsPK/Magma-Client-Facing-Final-Design?node-id=325%3A32572

### PR DESCRIPTION
**Pull request by HannaFar**
Signed-off-by: HannaFar <hannafarag159@gmail.com>

## Summary
 Add APN create/Edit Dialog.
 Add unit test

Capture d’écran 2020-10-22 à 21 43 59

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: nms/app/packages/magmalte/app/views/traffic/ApnEdit.js [hunk 1]
```diff
@@ -0,0 +1,352 @@
...
+        variant: 'success',
+      });
+      props.onSave(apn);
+    } catch (e) {
+      setError(e.response?.data?.message ?? e?.message);
+    }
+  };
+  return (
+    <>
+      <DialogContent data-testid="networkInfoEdit">
```
Comment (karthiksubraveti) [inline nms/app/packages/magmalte/app/views/traffic/ApnEdit.js]: minor nit: fix testId to match apn dialog

**Review by karthiksubraveti — COMMENTED** [review]
Given that we have only one field, I feel we can get rid of tabs and have just one form, what do you think ?

File: nms/app/packages/magmalte/app/views/traffic/ApnEdit.js [hunk 2]
```diff
@@ -0,0 +1,352 @@
...
+                  ...qosProfile,
+                  priority_level: parseInt(target.value),
+                })
+              }
+            />
+          </AltFormField>
+          <AltFormField label={'Max Required Bandwidth'}>
+            <OutlinedInput
+              data-testid="apnBandwidthUL"
+              placeholder="Enter Description"
```
Comment (andreilee) [inline nms/app/packages/magmalte/app/views/traffic/ApnEdit.js]: Change the placeholder. Give the input format, as well as the units

File: nms/app/packages/magmalte/app/views/traffic/ApnEdit.js [hunk 3]
```diff
@@ -0,0 +1,352 @@
...
+              value={qosProfile.class_id}
+              onChange={({target}) =>
+                setQosProfile({...qosProfile, class_id: parseInt(target.value)})
+              }
+            />
+          </AltFormField>
+          <AltFormField label={'ARP Priority Level'}>
+            <OutlinedInput
+              data-testid="apnPriority"
+              placeholder="Enter ID"
```
Comment (andreilee) [inline nms/app/packages/magmalte/app/views/traffic/ApnEdit.js]: Change the placeholder. Also specify allowable value range.

**Review by andreilee — COMMENTED** [review]
Some tooltips are likely required for each of the input fields. I believe this would all be self explanatory for a Magma expert, but not necessarily for others

**Review by karthiksubraveti — APPROVED** [review]
Changes LGTM.

**Pull request merged**